magento/values.yaml

```diff
@@ -0,0 +1,64 @@
+## Bitnami Magento image version
+## ref: https://hub.docker.com/r/bitnami/magento/tags/
+##
+imageTag: 2.0.5-r1
```

Comment:
> For some reason the image 2.0.5-r1 was not built. I have triggered the build and built it in any case.

I've seen this happen before. For some reason we need to manually register the `Docker` as a hook in the Github repo (or delete the DH repo and recreate it).